c10/test/util/optional_test.cpp

```diff
@@ -56,13 +58,6 @@ using OptionalTypes = ::testing::Types<
     // Non-trivial destructor.
     std::string>;
 
-// This assert is also in Optional.cpp; including here too to make it
```

Comment:
cc @ezyang if you have more context about this

Reply:
This is @swolchok's optimization, he knows ;)